single_chart_migration/migration.sh

```diff
@@ -47,17 +47,35 @@ cortex_nginx_deployment="pxcentral-cortex-nginx"
 ls_configmap="pxcentral-ls-configmap"
 post_install_job="pxcentral-post-install-hook"
 
+job_registry="docker.io"
+job_repo="portworx"
+job_image="pxcentral-onprem-post-setup"
+job_imagetag="1.3.0-dev"
```

Comment:
We need to remember to make the image tag to 1.3.0

Reply:
yes, maybe I will create another PR and keep it ready for changing these constants.